test/util/setup.js

```diff
@@ -0,0 +1,60 @@
+var loopback = require('loopback');
+
+var remoteConnector = require('../..');
+
+/**
+ * App helper
+ */
+function createRestAppAndListen(port) {
+  var app = loopback();
+
+  app.set('host', '127.0.0.1');
+  if (port) app.set('port', port);
+
+  app.use(loopback.rest());
+  app.locals.handler = app.listen();
+
+  return app;
+}
+module.exports.REST_APP = createRestAppAndListen;
```

Comment:
oh ok, I thought you were telling me to separate them hah... nvm then I'll get on that...